content/en/docs/tasks/administer-cluster/migrating-from-dockershim/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Migrating from dockershim to other container runtimes"
+weight: 10
+---
+
+<!-- overview -->
+
+This section presents information you need to know when migrating from
+dockershim to other container runtimes.
+
+Since the announcement of [dockershim deprecation] in Kubernetes 1.20, there
+were questions on how this will affect various workloads and Kubernetes
+installations. You can find these links useful to understand the problem better:
+
+- [Don't Panic: Kubernetes and Docker](/blog/2020/12/02/dont-panic-kubernetes-and-docker/)
+- [Dockershim Deprecation FAQ](/blog/2020/12/02/dockershim-faq/)
+
+It is recommended to migrate from dockershim to alternative container runtimes.
+Check out [container runtimes] section to know your options. Make sure to
+[report issues](https://github.com/kubernetes/kubernetes/issues) you encountered
+with the migration. So the issue can be fixed timely and your cluster would be
+ready for dockershim removal.
+
+[dockershim deprecation]: /blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation
```

Comment:
I'd suggest we avoid this style of markdown links because it is weird syntax for the link checker.
Please consider use inline links if you have another iteration.

Reply:
done